etc/bash_completion.d/singularity

```diff
@@ -369,9 +371,11 @@ _singularity() {
 
                 create)
                     if [[ ${cur} == -s || ${cur} == --size ]]; then
-                        _filedir
+                        return 0
+                    if [[ ${cur} == -i || ${cur} == --bytes-per-inode ]]; then
```

Comment:
Thars a syntax error here.  Needs to be an `elif`

Reply:
Why you care about syntax errors? It should **know** what I _mean_ ...